src/sass_context_wrapper.cpp

```diff
@@ -1,3 +1,4 @@
+#include <stdlib.h>
```

Comment:
It is standard ANSI C to include stdlib.h for malloc, and not platform specific. See http://en.m.wikipedia.org/wiki/C_dynamic_memory_allocation